devices/testarch-noffmux/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+generate_testarch_device_db(
```

Comment:
As we may want to generate different test architectures to test different features, we may want to put all of them under a `testarchs` directory?